extensions/amp-carousel/amp-carousel.md

```diff
@@ -72,35 +73,36 @@ Usage example:
 ```
 
 **type**
-- `carousel` (default) - All slides are shown and are scrollable horizontally.
-  Be aware that `type=carousel` does not currently support `layout=responsive`.
-- `slides` - Shows a single slide at a time. Supports `layout=responsive`.
+
+- `carousel` (default): All slides are shown and are scrollable horizontally.
+  The `carousel` type only supports `layout="fixed"` and `layout="fixed-height"`.
+- `slides`: Shows a single slide at a time. 
 
 **loop** (type=slides only)
 
 If present, the user may advance past the first item or the final item.
 
 **autoplay** (type=slides only)
 
-If present, advances the slide to the next slide without user interaction.
-By default it will advance a slide in 5000 millisecond intervals (5 seconds)
-and can be overridden by the `delay` attribute.
-If `autoplay` is present it will also attach the `loop` attribute to
-`amp-carousel` if `loop` is not already present.
+If present:
+
+- Advances the slide to the next slide without user interaction.
+By default, `autoplay` advances a slide in 5000 millisecond intervals (5 seconds); this can be overridden by the `delay` attribute.
+- Attaches the `loop` attribute to `amp-carousel` if `loop` is not already present.
 
 **delay** (type=slides only)
-By default a slide will advance in 5000 millisecond intervals (5 seconds)
+
+By default, a slide will advance in 5000 millisecond intervals (5 seconds)
 when `autoplay` is specified and will use the value of the `delay`
-attribute if present (minimum of 1000 ms; an error will be thrown if it's any lower).
-The value of `delay` must be a number of milliseconds, e.g. `delay=5000`.
+attribute if present (minimum of 1000 ms; an error will be thrown if it's any lower). The value of `delay` must be a number of milliseconds, e.g. `delay=5000`.
 
 ## Styling
 - You may use the `amp-carousel` element selector to style it freely.
 - You may use the `.amp-carousel-slide` class selector to target carousel items.
-- `.amp-carousel-button` by default uses an inlined SVG as the background-image of the buttons.
-You may override this with your own SVG or image like so:
+- By default, `.amp-carousel-button` uses an inlined SVG as the background-image of the buttons. You may override this with your own SVG or image as in the example below.
+- By default, the visual state of an `amp-carousel` button when it's disabled is hidden. You may override this visual state of an `amp-carousel` button by setting the visibility to `visible` as in the example below.
```

Comment:
@camelburrito - Do you want me to remove the last sentence in line 103 + remove the example?